eng/common/templates/job/source-index-stage1.yml

```diff
@@ -1,6 +1,6 @@
 parameters:
   runAsPublic: false
-  sourceIndexPackageVersion: 1.0.1-20210421.1
+  sourceIndexPackageVersion: 1.0.1-20210614.1
```

Comment:
You mentioned this is going to cause issues for arcade once it starts flowing again.  What action will need to be taken then?

Reply:
Putting it here means the next arcade update should stomp on this change and fix things. I'll watch it but things should be fine.